sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/Models/TelemetryItem.cs

```diff
@@ -95,12 +97,25 @@ public TelemetryItem(DateTime time, AzureMonitorResource? resource, string instr
             SetResourceSdkVersionAndIkey(resource, instrumentationKey);
         }
 
+        public void SetResourceCustomAttributes(AzureMonitorResource? _resource, IDictionary<string, string> properties)
+        {
+            if (_resource != null)
+            {
+                foreach (var attribute in _resource.CustomTags)
+                {
+                    properties.Add(attribute.Key, attribute.Value);
```

Comment:
If properties already contains the key .Add will throw an exception, if that is not a concern then this comment can be closed.

Unsure what the preferred value would be on a key collision.
If it is the _resource then I think this can be changed to 
```
                    properties[attribute.Key] = attribute.Value;
```
and remove the exception concern.

If the properties value should not be over written then need to check via Contains before Adding the value.

https://learn.microsoft.com/en-us/dotnet/api/system.collections.generic.idictionary-2.add?view=net-6.0#exceptions

Reply:
Has there been a precedent?  Last one in wins?  First one in wins?  I'll change to the proposed array index form and let's say last one in wins.